HIVE-21869 Clean up Kafka storage handler examples

kristopherkane opened the pull request.

b-slim commented on kafka-handler/README.md:
```diff
@@ -1,42 +1,71 @@
 |org.apache.hadoop.hive.serde2.JsonSerDe|
 |org.apache.hadoop.hive.serde2.OpenCSVSerde|
-|org.apache.hadoop.hive.serde2.avro.AvroSerDe|
+|org.apache.hadoop.hive.serde2.avro.AvroSerDe*|
```
not sure what is the `*` at the end

b-slim commented on kafka-handler/README.md:
```diff
@@ -1,42 +1,71 @@
 
-#### Table definition 
-In addition to the user defined payload schema Kafka Storage Handler will append additional columns allowing user to query the Kafka metadata fields:
+`*` This is just Apache Avro and not Confluent's serialization with schema registry integration.
```
nit...not sure if this is really needed.

b-slim commented on kafka-handler/README.md:
````diff
@@ -47,71 +76,139 @@ In addition to the user defined payload schema Kafka Storage Handler will append
 ```
 
-Count the number of records with Kafka record timestamp within the last 10 minutes interval.
+Count the number of records with where the record timestamp is within the last 10 minutes of query execution time.
````
with where does not sound correct to me

b-slim commented:
did quick pass and left minor comments. thanks for the contribution

kristopherkane commented:
Slim, I addressed all comments.  

Does the project prefer a JIRA patch over a GH PR? I did both for this one but subsequent JIRA only have patches.

b-slim commented:
Jira patch is a must because that is how we run tests.
But as you know we can not review a patch as is so that is why we need a GH PR.
Thus we need both, usually we start by GH PR to trigger the code review then when done we move to test the patch and make sure it passes all the tests.

b-slim commented:
LGTM thanks very much for this nice rewrite.

belugabehr commented:
Issue resolved through JIRA:

https://issues.apache.org/jira/browse/HIVE-21869
